app/src/main/res/values-nl/strings.xml

```diff
@@ -69,12 +83,18 @@
     <string name="action_discard_changes">Wijzigingen ongedaan maken</string>
     <string name="action_edit_playlist_rules">Bewerk filters</string>
     <string name="action_shuffle_all">Shuffle alles</string>
+    <string name="action_repeat_none">Niets herhalen</string>
+    <string name="action_repeat_one">Één lied herhalen</string>
+    <string name="action_repeat_all">Alles herhalen</string>
+    <string name="action_repeat_multi">Multi-herhalen&#8230;</string>
+
+    <!-- Confirmation messages -->
     <string name="confirm_disable_shuffle">Shuffle uitgeschakeld</string>
     <string name="confirm_enable_shuffle">Shuffle ingeschakeld</string>
     <string name="confirm_disable_repeat">Herhalen uitgeschakeld</string>
     <string name="confirm_enable_repeat">Herhalen ingeschakeld</string>
     <string name="confirm_enable_multi_repeat">Multi-herhalen (%d) ingeschakeld</string>
-    <string name="confirm_enable_repeat_one">Één keer herhalen ingeschakeld</string>
+    <string name="confirm_enable_repeat_one">Één lied herhalen ingeschakeld</string>
```

Comment:
Thanks for the context in your latest e-mail. `Oneindig herhalen ingeschakeld` is more appropriate.

Reply:
👍 Thank you!